cli/cli.go

```diff
@@ -345,6 +346,20 @@ documentation for detail information about writing tests.`,
 	}
 }
 
+func getTestGoExtesionsCommand() cli.Command {
+	return cli.Command{
+		Name:      "test_go_extensions",
+		ShortName: "test_go_ex",
```

Comment:
@p-kozlowski I plan to make this more intelligent and build the plugins only when necessary. Currently there is no such mechanism so the plugin is being built every time. According to the build process - this is how long the build process takes so I don't really think I'm able to do anything about it. But as I already said - the build process will be run only when necessary.